utils/run_test.sh

```diff
@@ -95,12 +96,11 @@ if [ -d "/mnt/daos" ]; then
             [ -z "$VALGRIND_SUPP" ] &&
                 VALGRIND_SUPP="$(pwd)/utils/test_memcheck.supp"
             VALGRIND_CMD="valgrind --leak-check=full --show-reachable=yes \
-                          --error-limit=no --suppressions=${VALGRIND_SUPP} \
-                          --xml=yes \
-                          --xml-file=memcheck-results-%p.xml"
+                       --error-limit=no --suppressions=${VALGRIND_SUPP} \
+                       --xml=yes \
+                       --xml-file=run_test_memcheck.sh/results-%p-memcheck.xml"
```

Comment:
You actually should line these up under the first line's options:
```suggestion
                                   --error-limit=no --suppressions=${VALGRIND_SUPP} \
                                   --xml=yes \
                                   --xml-file=run_test_memcheck.sh/results-%p-memcheck.xml"
```